.github/workflows/_container.yml

```diff
@@ -16,20 +16,7 @@ jobs:
         with:
           # Need this to get version number from last tag
           fetch-depth: 0
-      - name: Validate SemVer2 version compliance
-        if: startsWith(github.ref, 'refs/tags/')
-        env:
-          SEMVER_REGEX: ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$
 
-        run: |
-          ref="${{ github.ref_name }}"
-          my_regex="${{env.SEMVER_REGEX}}"
-          if [[ "$ref" =~ $my_regex ]]; then
-            echo "SemVer compliant version: $ref"
-          else
-            echo "Invalid SemVer version: $ref"
-            exit 1
-          fi
```

Comment:
As discussed offline, I would like proof we don't need this

Reply:
Given my recent mishap releasing something with the wrong tag, should we just put something like this into the `check` CI and have it only run on a tag, such that the entire pipeline fails early if someone tries to push an invalid tag?